lib/addressfinder/v2/au/batch_verification.rb

```diff
@@ -58,11 +58,12 @@ def verify_each_address_concurrently
         end
 
         # Verifies a single address, and writes the result into @results
-        def verify_address(address, index_of_address)
-          @results[index_of_address] =
-            AddressFinder::V2::Au::Verification.new(q: address, http: http.clone, **args).perform.result
+        def verify_address(address)
+          return if address.empty?
+
+          AddressFinder::V2::Au::Verification.new(q: address, http: http.clone, **args).perform.result || false
```

Comment:
both AU and NZ verification classes here will assign `result` only if the address is `matched`.
This means there's not way of knowing if an address is verified and has returned no result or if there's simply no address to verify.
Because of how we currently bulk verify addresses, an empty value within the column gets sent to the API, and results in an error within our reporting.